docs/fin/custom-commands.md

```diff
@@ -163,6 +163,61 @@ services:
       - VIRTUAL_HOST
 ```
 
+## Grouping Commands
+
+In certain cases you may want to group your commands so that you can commit them to a repository and easily share them with others, similar to a toolkit of commands that
+can be shared across a team of developers. Docksal allows for the ability to group these commands within a folder and are usable.
+
+When running `fin help` there should be similar output like below. This will show you the available commands.
+
+```
+Docksal control cli utility v1.54.4
```

Comment:
No need to include the whole `fin help` output, just the header and the custom commands section.